bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -171,6 +187,10 @@
 		<item-type>String</item-type>
 		<label>@text/channel-type.mielecloud.power_state_on_off.label</label>
 		<description>@text/channel-type.mielecloud.power_state_on_off.description</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Power</tag>
```

Comment:
See https://github.com/openhab/openhab-addons/pull/18585#discussion_r2055609847.

Reply:
Deleted the tags